src/app/public/modules/autocomplete/autocomplete.component.ts

```diff
@@ -206,7 +207,9 @@ export class SkyAutocompleteComponent
           this.closeDropdown();
         }
       });
+  }
 
+  public ngAfterViewInit(): void {
```

Comment:
Did changing this to `AfterViewInit` have an effect? We're using `AfterContentInit` because we're referencing a content child, `inputDirective` in the method.

Reply:
After our conversation, I'm going to revert this.